index.html

```diff
@@ -530,7 +530,7 @@ <h2>Flow Launcher is a free and open-source software</h2>
         localStorage.setItem("theme", theme);
 
     function toggle() {
-      const svgs = document.querySelectorAll("svg");
+      const svgs = document.querySelectorAll(".darkmode-btn svg");
```

Comment:
Without this, the class gets toggled on all SVGs on the page, include the icon for closing the Download dialog.